introduction.md

```diff
@@ -4,3 +4,9 @@ Fethi's Introduction
 Hello world!
 
 My favorite Code School path is iOS
+
+My deadly skills : 
+
+* Git.
+* JavaScript. 
+* Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?